src/PaymentBundle/Command/ExportCommand.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+namespace PaymentBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\Question;
+use Symfony\Component\Validator\Constraints\DateTime;
+
+class ExportCommand extends ContainerAwareCommand
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('mediaarea:payment:export')
+            ->setDescription('Export payment datas.')
+            ->setDefinition(array(
+                new InputArgument('start', InputArgument::REQUIRED, 'Start date (<info>YYYY-MM-DD HH:MM:SS</info>)'),
+                new InputArgument('end', InputArgument::REQUIRED, 'End date (<info>YYYY-MM-DD HH:MM:SS</info>)'),
+            ))
+            ->setHelp(
+                <<<'EOT'
+The <info>mediaarea:payment:export'</info> export payment datas within a date range
+Date format: <info>YYYY-MM-DD HH:MM:SS</info>
```

Comment:
Possible to have default values e.g. default start HH:MM:SS is 00:00:00 and default end  HH:MM:SS is 23:59:59.
Actually, a it is interesting per month, also just e.g. "mediaarea:payment:export 2017-09" would export invoices of September 2017.
also, ISO 8601 stipulate YYYY-MM-DDTHH:MM:SS avoiding the space between date and time.

Reply:
Done